src/platform/user/profile/vet360/containers/Vet360ProfileField.jsx

```diff
@@ -193,11 +193,7 @@ class Vet360ProfileField extends React.Component {
     };
 
     return (
-      <div
-        className="vet360-profile-field"
-        aria-atomic="false"
```

Comment:
I'd find it _super_ educational if you added a review comment here about why removing the `aria-atomic="false"` fixes the issue and/or why having it there created a problem.

Reply:
@erikphansen This is at the suggestion from @1Copenut

Reply:
sure, did you explain why this might work? cuz I have so much to learn about a11y stuff and this could be a good teaching moment :-D

Reply:
@erikphansen It's a hunch, TBH. We're hearing content on the page "behind the modal" repeated by IE11 + JAWS that really shouldn't be. I don't know the `aria-atomic` attribute is causing it, but wanted to remove it and rule it out before we spend more energy and time looking into deeper issues.